backend/internal/schema/hello/transactions/transactions.go

```diff
@@ -2,8 +2,9 @@ package hello
 
 import (
 	"github.com/gofiber/fiber/v2"
+	"github.com/jackc/pgx/v4"
 )
 
-func RetHelloWorld(ctx *fiber.Ctx) error {
+func RetHelloWorld(ctx *fiber.Ctx, db *pgx.Conn) error {
```

Comment:
by doing the struct field approach, you will resolve this error. also, is this really a "transaction"? i would consider this a handler